pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java

```diff
@@ -643,6 +644,14 @@ public int getConcurrency() throws SQLException {
   // TODO: In Java 8 this constant is missing, later versions (at least 11) have LocalDate#EPOCH:
   private static final LocalDate LOCAL_DATE_EPOCH = LocalDate.of(1970, 1, 1);
 
+  private @Nullable Instant getInstant(int i) throws SQLException {
+    OffsetDateTime odt = getOffsetDateTime(i);
```

Comment:
Similarly there is `timestamp.toInstant()`.  It isn't clear to me why going to OffsetDateTime is preferred when there is direct conversion both ways with java.sql.Timestamp.

Reply:
OffsetDateTime is very close to the wire format, so it makes sense to use it